frontend/packages/ceph-storage-plugin/integration-tests-cypress/support/vault-standalone.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  serviceAccountJSON,
+  roleBindingJSON,
+  getPVCJSON,
+  configMapJSON,
+  deploymentJson,
+  serviceJSON,
+  routeJSON,
+  networkPolicyJson,
+} from '../helpers/vault';
+
+export const configureVault = () => {
+  cy.exec('oc get project hashicorp', {
+    failOnNonZeroExit: false,
+  }).then(({ code }) => {
+    // Deploy vault only if doesn't already exist
+    if (code !== 0) {
+      cy.log('Create a new project for internel vault');
+      cy.exec('oc new-project hashicorp');
+
+      cy.log('Creating CR to configure vault');
+      cy.exec(`echo '${JSON.stringify(serviceAccountJSON)}' | oc apply -f -`);
+      cy.exec(`echo '${JSON.stringify(roleBindingJSON)}' | oc apply -f -`);
+      cy.exec(`echo '${JSON.stringify(getPVCJSON)}' | oc apply -f -`);
+      cy.exec(`echo '${JSON.stringify(configMapJSON)}' | oc apply -f -`);
+
+      cy.log('Deploying vault');
+      cy.exec(`echo '${JSON.stringify(deploymentJson)}' | oc apply -f -`);
+      cy.exec(`echo '${JSON.stringify(serviceJSON)}' | oc apply -f -`);
+      cy.log('Waiting for vault pod to be up and running');
+      cy.wait(50000);
```

Comment:
Deploying vault in the background, Need this wait to talk with vault server, Frontend check failure is expected

Reply:
perform polling on it.

Reply:
ack